keras_nlp/models/llama/llama_attention.py

```diff
@@ -17,35 +17,34 @@
 from keras_nlp.utils.keras_utils import clone_initializer
 
 
-class LlamaAttention(keras.layers.Layer):
-    """Grouped query attention for Llama models"""
+class CachedLlamaAttention(keras.layers.Layer):
```

Comment:
I don't actually mind `LlamaAttention` as a name here. I'd rather clean up the gemma side someday.

Reply:
Done.